src/lib/Command/IbexaSetupCommand.php

```diff
@@ -30,80 +33,217 @@ class IbexaSetupCommand extends BaseCommand
 
     private const PSH_RESOURCES_PATH = __DIR__ . '/../../../resources/platformsh';
 
+    private const DEFAULT_SOLR_PORT = 9000;
+
     protected function configure(): void
     {
         $this->setName('ibexa:setup')
              ->setDescription('Runs post install configuration tool.')
              ->addOption('platformsh', null, InputOption::VALUE_NONE, 'Install Platform.sh config files')
+             ->addOption('solr', null, InputOption::VALUE_OPTIONAL, 'Setup SOLR for development purposes', self::DEFAULT_SOLR_PORT)
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         if ($input->getOption('platformsh')) {
-            $this->getIO()->write('Installing Platform.sh config files...', true, IOInterface::NORMAL);
+            $this->installPlatformShConfigFiles($input, $output);
+        }
+        if ($input->getOption('solr')) {
+            $this->installSolrConfigFiles($input, $output);
+        }
 
-            $fileSystem = new Filesystem();
+        return 1;
```

Comment:
Did you mean `return self::SUCCESS` ? Exit code `1` means that command execution failed.

Reply:
That what was in the original command. I had now reverted it whole back and changed return code.